tests/trainer/connectors/test_signal_connector.py

```diff
@@ -57,3 +58,16 @@ def training_step(self, batch, batch_idx):
         else:
             trainer.fit(model)
         assert trainer._terminate_gracefully == (False if register_handler else terminate_gracefully)
+
+
+def _registering_signals():
+    with mock.patch.dict(os.environ, {"PL_FAULT_TOLERANT_TRAINING": "1"}):
+        trainer = Trainer()
+        trainer.signal_connector.register_signal_handlers()
+
+
+@RunIf(skip_windows=True)
+def test_signal_connector_in_thread():
```

Comment:
Shouldn't this set the fault tolerant env variable?

Reply:
Done !